test/e2e/onLCP-test.js

```diff
@@ -717,7 +718,10 @@ const assertFullReportsAreCorrect = (beacons) => {
 
   // Temp fix to address Firefox flakiness.
   // See https://github.com/GoogleChrome/web-vitals/issues/472
-  if (browser.capabilities.browserName !== 'firefox') {
+  if (
```

Comment:
I think this logic is incorrect.

Right now I think it will only run the assertion locally in Firefox.

Reply:
Fixed